incalmo/c2server/c2server.py

```diff
@@ -244,6 +245,60 @@ def agent_download():
         return jsonify({"error": f"Internal server error: {str(e)}"}), 500
 
 
+# Stream logs
+@app.route("/stream_logs", methods=["GET"])
```

Comment:
Rather than a stream this might be easier to program as a normal GET request the client polls or a Websocket (kinda heavy weight)